etc/listr2.api.md

```diff
@@ -160,7 +160,7 @@ export interface ListrOptions<Ctx = ListrContext> {
 
 // @public
 export class ListrRenderer {
-    constructor(tasks: readonly ListrTaskObject<any, ListrRendererFactory>[], options: typeof ListrRenderer.rendererOptions, renderHook$?: Subject<void>);
+    constructor(tasks: readonly ListrTaskObject<any, ListrRendererFactory>[], options: typeof ListrRenderer.rendererOptions | undefined, renderHook$?: Subject<void>);

```

Comment:
Backwards compatible because this argument could always have been undefined

Reply:
I set it to never when there are no renderer options, isn't it better to keep at is it is?